rskj-core/src/main/java/co/rsk/scoring/PeerScoring.java

```diff
@@ -16,34 +36,38 @@
  * Created by ajlopez on 27/06/2017.
  */
 public class PeerScoring {
+    private static final Logger logger = LoggerFactory.getLogger("peerScoring");
+
+    private final String peerKey;

```

Comment:
`peerKey` confuses a bit. Do you think it could be `nodeId` or `peerId`?

Reply:
`peerKey` could be `nodeId` or `address`, we record scoring by both fields.
And I thought `peerId` could be confused with `nodeId`, that's why I've chosen `peerKey`.
But rethinking it... I think `peerId` is not so bad 👍

Reply:
Done